data_preprocessing/data_preprocessor.py

```diff
@@ -28,6 +28,14 @@ def choose_columns(self, columns: list[str]):
     def transform_column_data_to_logarithmic_scale(self, column: str):
         pass
 
+    @abstractmethod
+    def scaling_column_data_numerical_attributes(self, column: str):
+        pass
```

Comment:
Is this function used somewhere?

Reply:
Yes, unfortunately due to this problem with 'No module named...' while trying to run main.py, I had to test it on Google Colab :P